packages/client/components/developers/DeveloperList.tsx

```diff
@@ -18,12 +30,7 @@ export default function DeveloperList() {
         mx="auto"
         p={10}
       >
-        <DeveloperItem />
-        <DeveloperItem />
-        <DeveloperItem />
-        <DeveloperItem />
-        <DeveloperItem />
-        <DeveloperItem />
+        {devs && devs.map((dev) => <DeveloperItem key={dev.id} {...dev} />)}
```

Comment:
Very trivial comment, but might look a little bit cool

Suggested change for this specific line is
```tsx
{(devs || []).map((dev) => <DeveloperItem key={dev.id} {...dev} />)}
```

Reply:
What's the logic behind this vs @PBillingsby's? Just curious

Reply:
On the current implementation, it checks if `devs` is not `undefined` or `null`, then render the `devs.map()`

On the suggested change, is a bit the same. It looks at `devs` and if that is not `undefined` or `null` then it will render `devs.map()`, but when it is `undefined` or `null` it returns an empty array `[]`. The `devs.map()` still gets called but since it being an empty array, it won't render those `<DeveloperItem />` components

Reply:
Actually just no important change here, but just wanted to share some thoughts or just want to comment in general :P 

Both already achieved what it needs to get the job done, so you can omit my comment @PBillingsby and no further changes needed